src/libstd/sys/windows/args.rs

```diff
@@ -48,14 +176,13 @@ impl<'a> fmt::Debug for ArgsInnerDebug<'a> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         f.write_str("[")?;
         let mut first = true;
-        for i in self.args.range.clone() {
+        for i in self.args.parsed_args_list.clone() {
```

Comment:
Instead of `clone` can this use `IntoIter::as_slice`? Perhaps this could simply delegate to that?